package/pyproject.toml

```diff
@@ -44,6 +44,7 @@ dependencies = [
     'mda-xdrlib',
     'waterdynamics',
     'pathsimanalysis',
+    'mdahole2',
```

Comment:
This is going to be annoying to package upstream, but it's a "me" problem.

Reply:
Because you need to make hole2 available in spack?

Reply:
I thought so, but looking at [mdahole2](https://github.com/MDAnalysis/mdahole2)'s dependencies it does not look like hole2 is provided, is it? So packaging mdahole2 would be easier, although I don't think it would make much sense to provide a downstream Spack package without hole2 TBH.

Reply:
We have a hole2 cf package https://anaconda.org/conda-forge/hole2 for Linux and osx — just not for Windows.

Reply:
We didn't include it as a dependency because ... well, because we only got the conda-forge package recently and used the reasoning for the pip package (see also https://github.com/MDAnalysis/mdahole2/issues/25 ) that people should be able to use their own hole2 installation (as it worked previously).

Reply:
See https://github.com/MDAnalysis/mdahole2/issues/55

Reply:
Just to clarify the current situation here:

1. PyPi mdahole2 / pure pip install - has no hole2 dependency. This is because we can't distribute hole2 over pypi at the moment.
2. conda-forge mdahole2-base: has no hole2 dependency - this exists for all the os & architecture types where hole2 can't be used. The combination of MDAnalysis & mdahole2-base effectively replicates the current status of the MDA core library where we ask folks to install hole2 themselves.
3. conda-forge mdahole2: for x86_64 linux and macos this is mdahole2-base + hole2

Reply:
Thanks for explaining. I won't resolve the comment, just so that it remains visible.